less/about/siteDetails.less

```diff
@@ -82,6 +82,9 @@ body {
       font-size: 16px;
       margin-bottom: 12px;
       padding-left: @aboutPageSectionPadding;
+      width: 400px;
+      display: flex;
+      justify-content: space-between;
```

Comment:
ah i see now. Sorry abt that I dint realize we lost the color of the title. Will make the change tonight.